README.md

```diff
@@ -98,42 +112,36 @@ Whether or not to enable SSL communication.
 * Valid options: [ `true`, `false` ]
 * Default: `false`
 
-##### `verify_ssl`
+##### `ssl_verify`
 Whether or not to verify the SSL CA/Peer on the certifcate. Set to false if using a self-signed certificate and the CA is not installed locally.
```

Comment:
This is a very bad approach. SSL should always be verified. Instead of defaulting to false, we should enable the user to provide the CA file.

Reply:
It should definitely, but even in my own company (which is an MSSP), it tends to get ignored in internal-only server-server communication. 

Add support for providing the CA will be added in the future, maybe we can revisit then.